spec/integration/magic_variables_spec.rb

```diff
@@ -77,6 +77,19 @@
     end
   end
 
+  context '$_root' do
+    it 'evaluates to the root path of the repository' do
+      GitshRunner.interactive do |gitsh|
+        root_dir = Dir.pwd
+        gitsh.type('init')
+        gitsh.type(':echo $_root')
```

Comment:
Given this setup, this is the same as the working directory. I'd love for this test to start gitsh from a directory where the parent dir has a`.git` instead.